### PR TITLE
Adding lando open command.

### PR DESCRIPTION
**Pull request by serundeputy**

### CODE REVIEW

File: docs/cli/open.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Usage
+-----
+
+```bash
+# Opens a reasonable guess at your default url
+lando open
+
+# Opens a specific URL
+lando open --url https://myapp.lndo.site
```
Comment (uberhacker) [inline docs/cli/open.md]: Is the `--url` switch necessary?  I'm even lazier than you think. ;)

Comment (uberhacker) [reply docs/cli/open.md]: Right, but do you even need it with the override?  Can't you just use the domain as the argument?

Comment (serundeputy) [reply docs/cli/open.md]: Ahh; I see; you are probably right; I'll check

Comment (pirog) [reply docs/cli/open.md]: @serundeputy dropping a note here: might want to test this against non-recipes to get a sense of the issues we are up against

**Comment by dustinleblanc** [comment]
Do we want to defer this till later or try to get it in this release?

**Comment by pirog** [comment]
@dustinleblanc i think defer. Specifically i thin we need to figure out what to do re: https://github.com/lando/lando/issues/447#issuecomment-355786058 first

**Comment by dustinleblanc** [comment]
Since we're deferring this, I am closing the PR

**Pull request closed without merging**